code/campaignclassic/src/main/res/layout/push_template_auto_carousel.xml

```diff
@@ -15,18 +18,26 @@
         android:id="@+id/notification_title"
         style="@style/Title"
         android:layout_width="match_parent"
-        android:layout_height="20dp"
+        android:layout_height="wrap_content"
```

Comment:
The docs advice against using `wrap_content` with auto sizing text views as it can cause unexpected behavior. can we revert to using hard coded values?

Reply:
The space provided within the template remote view is limited and `wrap_content` is needed to ensure that a text view is collapsed if it contains limited text. I've removed the auto sizing of text as the display space would never allow larger sizes other than the default.